packages/repository/src/repositories/repository.ts

```diff
@@ -303,7 +303,11 @@ export class CrudRepositoryImpl<T extends Entity, ID>
     );
   }
 
-  async findById(id: ID, filter?: Filter<T>, options?: Options): Promise<T> {
+  async findById(
+    id: ID,
+    filter?: FilterExcludingWhere<T>,
+    options?: Options,
+  ): Promise<T> {
```

Comment:
>
> To clarify this for developers, maybe we can add this rationale here?:
>

Should be doable as such:

```suggestion
  /**
   * {@inheritDoc EntityCrudRepository.findById}
   */
  async findById(
    id: ID,
    filter?: FilterExcludingWhere<T>,
    options?: Options,
  ): Promise<T> {
```

See: https://api-extractor.com/pages/tsdoc/tag_inheritdoc/

Reply:
I added tsdocs to the base interface.